src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,57 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private static final int MAX_VALUE = 11;
```

Comment:
This constant is related to CAPACITY . Changing only one leads to an error (inconsistency).